apps/meteor/client/components/UserAndRoomAutoCompleteMultiple/UserAndRoomAutoCompleteMultiple.tsx

```diff
@@ -51,6 +51,7 @@ const UserAndRoomAutoCompleteMultiple = ({ value, onChange, ...props }: UserAndR
 	return (
 		<AutoComplete
 			{...props}
+			data-qa-type='user-and-room-auto-complete-input'
```

Comment:
```suggestion
			name='userAndRoom'
```
Can we avoid adding this kind of prop?
Instead, you can pass a name for this component and get it by role and this name

e.g: `page.getByRole('textbox', { name: 'userAndRoom' })`